.travis/run.sh

```diff
@@ -22,7 +22,11 @@ fi
 
 source ~/.venv/bin/activate
 
-if [ -n "${TOXENV}" ]; then
+if [ -n "${DOCKER}" ]; then
+    # We will be able to drop the -u once we switch the default container user in the
+    # dockerfiles.
+    docker run -u 2000:2000 -v "${TRAVIS_BUILD_DIR}":"${TRAVIS_BUILD_DIR}" -v "${HOME}/wycheproof":/wycheproof -e TOXENV "${DOCKER}" /bin/sh -c "cd ${TRAVIS_BUILD_DIR};tox -- --wycheproof-root='/wycheproof'"
```

Comment:
Can you add some line breaks so this is readable

Reply:
Precious bytes.

Reply:
You know I love wantonly wasting bytes